### PR TITLE
[11.0][MIG] partner_risk_insurance

### PR DESCRIPTION
**Pull request by sergio-teruel**
cc @Tecnativa

### CODE REVIEW

File: partner_risk_insurance/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "Partner Risk Insurance",
+    "version": "11.0.1.0.0",
+    "development_status": "Production/Stable",
+    "summary": "Risk insurance partner information",
+    "author": "OdooMRP team,"
```
Comment (pedrobaeza) [inline partner_risk_insurance/__manifest__.py]: Remove this line

File: partner_risk_insurance/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+              "AvanzOSC,"
+              "Tecnativa - Pedro M. Baeza,"
```
Comment (pedrobaeza) [inline partner_risk_insurance/__manifest__.py]: Put only `Tecnativa`

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

File: partner_risk_insurance/README.rst [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+
+Credits
+=======
+
+Authors
+~~~~~~~
+
+* OdooMRP team
+* AvanzOSC
+* Tecnativa - Pedro M. Baeza
```
Comment (rafaelbn) [inline partner_risk_insurance/README.rst:60]: Is this line to be recreated with this new value? https://github.com/OCA/credit-control/pull/7/files#diff-0f1fbcd1c48ec7fa07c7af50d7d6d1c1R8

**Review by rafaelbn — APPROVED** [review]
Tested functionally in runbot :+1: 

Ref of Customer
http://3337647-7-f3a20a.runbot2.odoo-community.org/web#id=13&view_type=form&model=res.partner

File: partner_risk_insurance/README.rst (hunk 3 above)
Comment (LoisRForgeFlow) [reply partner_risk_insurance/README.rst:60]: Yes, it will be changed after merge.

**Review by LoisRForgeFlow — APPROVED** [review]
Code review LGTM

**Comment by LoisRForgeFlow** [comment]
@pedrobaeza if you update your review, this one is ready.

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**